src/objects.rs

```diff
@@ -18,9 +19,9 @@ pub enum ObjectShape {
 }
 
 #[derive(Debug, PartialEq, Clone)]
-pub struct Object {
+pub struct ObjectData {
     pub id: u32,
-    pub gid: u32,
+    tileset_idx_tile_id: Option<(u32, u32)>,
```

Comment:
Oh, perfect then, wasn't using it because I didn't know that flipping was possible here.

Reply:
Where's this in the docs, BTW? I can only find the `gid` attribute but not anything related to flipping in https://doc.mapeditor.org/en/stable/reference/tmx-map-format/#tmx-object.

Reply:
I guess it's kind of implicit, and it would be good to explicitly say that because this is a GID, it might also include its [flipping flags](https://doc.mapeditor.org/en/stable/reference/global-tile-ids/#tile-flipping).